MontyTRC89/TombEngine
# Sound engine enhancements

* Introduce new audio track channel for playing voiceovers. It will stop not only when game is paused, but also when it is in inventory.
* If subtitle file with same name as voiceover track exists in audio folder or audio/Subtitles folder, it is loaded and can be used with `GetCurrentSubtitle` script function.
* Introduce loudness measurement script function which can be used for voiceovers or audio visualization.
* Introduce `IsAudioTrackPlaying` script function.
* Don't stop ambience track when Lara dies.
* Fix epic bug with string pointer corruption during audio engine initialization.

To test:

* New audio track channel - how it behaves (pause, inventory, load/save, switching different tracks etc)
* Subtitles for voice track channel
* How `GetAudioTrackLoudness` and `IsAudioTrackPlaying` works
* Make sure no new audio bugs are introduced
